core/state/prefeth.go

```diff
@@ -0,0 +1,199 @@
+package state
+
+import (
+	"bytes"
+	"github.com/ethereum/go-ethereum/common"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/ethereum/go-ethereum/trie"
+	"log"
+	"math/big"
+	"sync"
+	"sync/atomic"
+	"time"
+)
+
+type prefetchJob struct {
+	accountAddr []byte
+	account     *Account
+	start, end  []byte
+}
+
+// Prefetch If redis is empty, the hit rate will be too low and the synchronization block speed will be slow
+// this function will parallel load the latest block statedb to redis
+func (s *DB) Prefetch(parallel int) {
+	wg := sync.WaitGroup{}
+
+	jobChan := make(chan *prefetchJob, 10000)
```

Comment:
why 10000 ? can be replaced by a constant

Reply:
this function used by debug or first time to init tikv cluster, 10000 is the relatively balanced value tested